mmv1/products/dataplex/Datascan.yaml

```diff
@@ -0,0 +1,781 @@
+    output: true
+    description: |
+      System generated globally unique ID for the scan. This ID will be different if the scan is deleted and re-created with the same name.
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      Description of the scan.
+  - !ruby/object:Api::Type::String
+    name: 'displayName'
+    description: |
+      User friendly display name.
+  - !ruby/object:Api::Type::KeyValuePairs
+    name: 'labels'
+    description: |
+      User-defined labels for the scan. A list of key->value pairs.
+  - !ruby/object:Api::Type::Enum
+    name: 'state'
+    output: true
+    description: |
+      Current state of the DataScan.
+    values:
+      - :STATE_UNSPECIFIED
+      - :ACTIVE
+      - :CREATING
+      - :DELETING
+      - :ACTION_REQUIRED
+  - !ruby/object:Api::Type::String
+    name: 'createTime'
+    output: true
+    description: |
+      The time when the scan was created.
+  - !ruby/object:Api::Type::String
+    name: 'updateTime'
+    output: true
+    description: |
+      The time when the scan was last updated.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'data'
+    required: true
+    immutable: true
+    description: |
+      The data source for DataScan.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'entity'
+        immutable: true
+        exactly_one_of:
+          - data.0.entity
+          - data.0.resource
+        description: |
+          The Dataplex entity that represents the data source(e.g. BigQuery table) for Datascan.
+      - !ruby/object:Api::Type::String
+        name: 'resource'
+        immutable: true
+        exactly_one_of:
+          - entity
+          - resource
+        description: |
+          The service-qualified full resource name of the cloud resource for a DataScan job to scan against. The field could be:
+          (Cloud Storage bucket for DataDiscoveryScan)BigQuery table of type "TABLE" for DataProfileScan/DataQualityScan.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'executionSpec'
+    required: true
+    description: |
+      DataScan execution settings.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: 'trigger'
+        required: true
+        description: |
+          Spec related to how often and when a scan should be triggered.
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: 'onDemand'
+            allow_empty_object: true
+            send_empty_value: true
+            exactly_one_of:
+              - execution_spec.0.trigger.0.on_demand
+              - execution_spec.0.trigger.0.schedule
+            description: |
+              The scan runs once via dataScans.run API.
+            properties: []
+          - !ruby/object:Api::Type::NestedObject
+            name: 'schedule'
+            description: |
+             The scan is scheduled to run periodically.
+            exactly_one_of:
+              - onDemand
```

Comment:
similar to 179 & 180 above, these will need the `execution_spec.0.trigger.0.` prefix

Reply:
Done